### ISSUE TITLE
Town Hall February 19th 2pm ET

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/bootstrap#105]
Hey All,

We are going to be holding an open town hall tomorrow, February 19th, at 2 p.m. ET to discuss the current state of the Node.js Foundation js + JS Foundation merger.

Please add questions to our [google form](https://goo.gl/forms/1IXNT9M8EUNgTyV32)

To Join:

https://zoom.us/j/362277753

Or iPhone one-tap :
US: +16699006833,,362277753# or +16465588656,,362277753#
Or Telephone:
Dial(for higher quality, dial a number based on your current location):
US: +1 669 900 6833 or +1 646 558 8656 or +1 877 369 0926 (Toll Free) or +1 855 880 1246 (Toll Free)
Webinar ID: 362 277 753
International numbers available: https://zoom.us/u/axkB6o3Eo

**Comment by MylesBorins** [comment]
Some TL;DR notes about current status from #104

---

Community Board Reps to stage 2: #103
CPC Charter to stage 2: #100
Individual Membership to stage 2: #102

TL;DR (lmk if I missed anything important)

**Community Board Reps**

* 1 board seat for CPC
* 1 floating board seat
  
  * process for seat determined annually by CPC
  * requires 2 Platinum board members to activate
* 1 board seat for individual membership program
  
  * requires 2 platinum board members and 2000 program members to activate

**CPC**

* CPC is made up of
  
  * Voting members (2 members per Impact Stage project, 2 for non-Impact projects, and 2 nominated by
    regular members)
  * Regular members (any member of any foundation project an get added to list of regular members)
  * Observers (any member of any foundation project)
* All membership types can attend any meeting and participate in consensus process
* If there is a vote only voting members participate
* CPC has a chair and director (director attends board meetings)
* CPC has a second board seat that is allocated on an annual basis
* CPC sets foundation wide policy
* CPC is final authority to decide if projects join foundation
* CPC does not have control of technical direction of projects within the foundation
* CPC oversees budget for foundation projects
* CPC will charter a team to help with CoC escalations within foundation.

**Individual Membership**

* CPC will be chartered by board with administering the program (in bylaws)
* Program will have board seat that activates when:
  
  * there are 3 platinum directors
  * there are 2000 members registered in the program
* Program will initially have one membership type but will expand to 3 if program is successful

**Comment by codeekage** [comment]
Typo 

Fixed 
> Regular members (any member of any foundation project an get added to list of regular members)

To 

> Regular members (any member of any foundation project can get added to list of regular members)
